### ISSUE TITLE
Query tags by verse reference

### ISSUE THREAD

**Issue by chisomosteve** [bibletag/bibletagapi#15]
At the moment is appears you can only query the actual tag value.
It would be very helpful to find all the tags connected to a verse reference (or even multiple verses).

**Comment by dwhitena** [comment]
Yes!  Definitely agree with this.  Why don't we also add all verses connected to a particular tag.

**Comment by chisomosteve** [comment]
Doesn't it do that already? I thought that's what the Angular front-end did?

**Comment by dwhitena** [comment]
@chisomosteve It return a single verse for a given tag.  However, I don't have it built out to return all tagged verses for a given tag.

**Comment by dwhitena** [comment]
@chisomosteve That is, right now it just returns the top ranked verse for a given tag.
